api/callbacks.go

```diff
@@ -73,10 +73,30 @@ func recoverPanic(ret *C.GoResult) {
 	}
 }
 
+// GasMultiplier is how many cosmwasm gas points = 1 sdk gas point
+// SDK reference costs can be found here: https://github.com/cosmos/cosmos-sdk/blob/02c6c9fafd58da88550ab4d7d494724a477c8a68/store/types/gas.go#L153-L164
+// A write at ~3000 gas and ~200us = 10 gas per us (microsecond) cpu/io
+// Rough timing have 88k gas at 90us, which is equal to 1k sdk gas... (one read)
+const GasMultiplier = 100
```

Comment:
This is copied over from [a definition in wasmd](https://github.com/CosmWasm/wasmd/blob/da3964a78622470e326089a548695ff0bfcc1793/x/wasm/internal/keeper/keeper.go#L25)
The definition in wasmd should be replaced with an import of this definition.

Reply:
I think it might be possible if you had a submodule with a hidden global, a boolean to prevent setting it more than once, a mutex, a setter function, and a getter function. You could have the 100 as the default value. I don't really know Go well enough to implement this, so if you really want this i'll leave it up to you.

Reply:
:boom: 

Actually mutable globals is easy in Go. And very easy to create race conditions and other bad behavior. SInce the abci (cosmos sdk) app is single threaded, we could just expose a global var and they could write it and it would be safe (but ugly).

Maybe you can just change this from `const` to `var` now and we can worry about the "proper" way to do it when there is a multi-threaded Go runtime on top of this. (With a big comment explaining how this is unsafe and Ethan made you do it :wink: )